doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -280,6 +280,20 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
 
+.. rst-class:: v2-4-1 v2-4-0
+
+KRKNWK-17699: Non complaiant Basic Information cluster revision for Matter 1.1.0. The issue result in fail at TC-BINFO-1.1 Matter Certifiction test.
+
+  **Workaround:** Set the default value of the Basic Information cluster revision to 1 in ZAP and regenerate files.
+
+
+.. rst-class:: v2-4-1 v2-4-0 v2-3-0 v2-2-0
+
+KRKNWK-17594: Application may not response for the forcing arm-fail-safe cleanup due to unecessery reset Thread interface. The impact may result in fail at TC-CNET-4.10 Matter Certifiction test.
```

Comment:
```suggestion
KRKNWK-17594: Application does not always respond when forcing arm-fail-safe cleanup because of the Thread interface being unnecessarily reset.
  This can result in the TC-CNET-4.10 Matter Certification test failing.
```
A couple of points:
* What is "arm-fail-safe cleanup"? It sounds like something ARM-based has failed a safe cleanup.
* The way I worded it means that the forcing the cleanup happens because the Thread interface was reset without reason, but it could also be that it should be an unexpected reset of the Thread interface or that the not responding happens due the unnecessary reset of the Thread interface.

Reply:
* arm-fail-safe is an procedure defined by Matter Specification.
* not responding happens due to unnecessary reset of the Thread Interface

Reply:
```suggestion
KRKNWK-17594: Application does not always respond when forcing fail-safe cleanup.
  This happens because of the Thread interface being unnecessarily reset, and can result in the TC-CNET-4.10 Matter Certification test failing.
```